server/__tests__/unit/campaigns.test.js

```diff
@@ -0,0 +1,29 @@
+// server/__tests__/api/campaigns.test.js
+const request = require('supertest')
+const app = require('../../app')
+const Campaign = require('../../db/models/campaign')
+
+jest.mock('../../db/models/campaign', () => {
+  return {
```

Comment:
Nice 😎

Reply:
Thank you for the review. I have made and pushed all appropriate changes as you mentioned.